.github/workflows/check-documentation.yml

```diff
@@ -39,4 +39,5 @@ jobs:
           echo "This can happen if you haven't run 'go generate' after making a change."
           echo "Please run go generate and commit the documentation changes to resolve this."
           echo "Changed files: ${{ steps.verify-changed-files.outputs.changed_files }}"
+          git diff
```

Comment:
Is this intentional?

Reply:
Yes, it's a bit difficult to understand the exact change in a file unless you diff the whole thing. So this is a convenience thing.

Reply:
I had this case where I haven't rebased but the CI test was running against the virtual merge commit, so it was showing me a diff but locally I had none. But since it wasn't telling me what I missed, I had a hard time debugging it until I added an explicit `git diff`. At which point I thought that it actually belongs here.

Reply:
Sounds reasonable.